src/account/multiple.js

```diff
@@ -134,17 +135,17 @@ export default AccountBase.compose(AsyncInit, {
      */
     _resolveAccount (account) {
       if (account === null) {
-        throw new Error('No account or wallet configured')
+        throw new UnknownAccountError()
```

Comment:
The logic here is broken by previous developers, if I accidentally do
```
sdk.someMethod({ onAccount: null })
```
then SDK would say that I don't have accounts at all?